services/movie.py

```diff
@@ -0,0 +1,41 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list | None = None,
+        actors_ids: list | None = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids is None and actors_ids is None:
+        return queryset
+
+    if genres_ids and actors_ids is None:
+        queryset = queryset.filter(genres__in=genres_ids)
+        return queryset
+
+    if actors_ids and genres_ids is None:
+        queryset = queryset.filter(actors__in=actors_ids)
+        return queryset
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(genres__in=genres_ids).filter(
+            actors__in=actors_ids)
+        return queryset
```

Comment:
You can simplify this method
```suggestion
    if genres_ids:
        queryset = queryset.filter(genres__in=genres_ids)
    if actors_ids:
        queryset = queryset.filter(actors__in=actors_ids)
    return queryset
```